common/src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java

```diff
@@ -423,9 +430,61 @@ public class GTRecipeTypes {
             })
             .setSound(GTSoundEntries.FURNACE);
 
-    // TODO Add small distillery recipes in .onRecipeBuild()
     public final static GTRecipeType DISTILLATION_RECIPES = register("distillation_tower", MULTIBLOCK).setMaxIOSize(0, 1, 1, 12).setEUIO(IO.IN)
-            .setSound(GTSoundEntries.CHEMICAL);
+            .setSound(GTSoundEntries.CHEMICAL)
+            .onRecipeBuild((recipeBuilder, provider) -> {
+                if (recipeBuilder.output.containsKey(FluidRecipeCapability.CAP)) {
+                    long EUt = EURecipeCapability.CAP.of(recipeBuilder.tickInput.get(EURecipeCapability.CAP).get(0).getContent());
```

Comment:
les add a flag for `//disabledDistilleryRecipes() TODO` in DISTILLATION_RECIPES.

add a method `disableDistilleryRecipes` in GRRecipeBuilder, to add a flag by adding data.
```java
public GTRecipeBuilder disableDistilleryRecipes(boolean flag) {
    return addData("disable_distillery", false);
}
```
```java
DISTILLATION_RECIPES.recipeBuilder("xx")
    ......
    .disableDistilleryRecipes()
```
```java
.onRecipeBuild((recipeBuilder, provider) -> {
    if (builder.data.getBoolean("disable_distillery")) return;
    ....
})
```